go.mod

```diff
@@ -0,0 +1,17 @@
+module github.com/buildpacks/profile
+
+go 1.17
```

Comment:
any reason to not use 1.18?

Reply:
I don't have a strong preference. On the Paketo Java buildpacks we're on 1.17, so I just used that out of habit. Our rationale is that we don't want to run the absolute latest version of Go, we don't often need those features and don't want any potential bugs/issues, so we hang back a release.

Reply:
👍 , makes sense. I dont have the power to approve and merge but this looks like a sensible template to build upon.